delta/layers/ops/kernels/delta_delta.cc

```diff
@@ -88,7 +88,7 @@ void DeltaDelta::Compute(const Tensor& input_feats, int frame,
   output->resize(output_dim);
   auto input = input_feats.matrix<float>();
 
-  for (int i = 0; i < order_; i++) {
+  for (int i = 0; i <= order_; i++) {
```

Comment:
why change the `order_`

Reply:
Otherwise, there will not make the last delta.